payment-request/allowpaymentrequest/removing-allowpaymentrequest.https.sub.html

```diff
@@ -13,7 +13,7 @@
   const path = location.pathname.substring(0, location.pathname.lastIndexOf('/') + 1);
   iframe.src = "https://{{domains[www1]}}:{{ports[https][0]}}" + path + "echo-PaymentRequest.html";
   iframe.onload = t.step_func(() => {
-    if (i === 0) {
+    if (i === 1) {
```

Comment:
@AmazingJaze thanks for the clarification. 

@zcorpan (and @sideshowbarker, assuming you reviewed tis), we should avoid tests like this - it's too easy to screw them up for both the reviewer and the person writing them.